### ISSUE TITLE
Update packages to avoid using undocumented kernel function names `(AINV|INV|ONE|ZERO)_MUT`

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#4762]
Following up PR #4761, we should replace use of the following internal names for various kernel operations by their official names (they actually all have at least two...), so that we are free to eventually replace them with less confusing ones (note how the `_MUT` suffix has a different meaning from AINV/ZERO to INV/ONE):
- `AINV` -> `AdditiveInverseSameMutability` or `AdditiveInverseSM`
- `AINV_MUT` -> `AdditiveInverseMutable` or `AdditiveInverseOp`
- `INV` -> `InverseMutable` or `InverseOp`
- `INV_MUT` -> `InverseSameMutability` or `InverseSM`
- `ONE` -> `OneMutable` or `OneOp`
- `ONE_MUT` -> `OneSameMutability` or `OneSM`
- `ZERO` -> `ZeroSameMutability` or `ZeroSM`
- `ZERO_MUT` -> `ZeroMutable` or `ZeroOp`
- (as a bonus, it'd also be good to replace `SUM` -> `\+`, `DIFF` -> `\-`, `PROD` -> `\*`, `QUO` -> `\/`, `LQUO` -> `LeftQuotient`, `POW` -> `\^`, `LT` -> `\<`, `EQ` -> `\=`)

Affected distributed packages that need a fix and release (I'll leave the final checkmark for when the packages are picked up by the distribution mechanism):
- [x] corelg
  - [x] patch merged?
  - [x] released in version 1.55
- [x] float
  - [x] https://github.com/gap-packages/float/pull/80 merged?
  - [x] released in version 1.0.3
- [x] fr
  - [x] https://github.com/gap-packages/fr/pull/46 merged?
  - [x] released in version 2.4.8
- [x] Gauss
  - [x] patch merged?
  - [x] released in version 2022.02-01
- [x] guarana
  - [x] patch merged?
  - [x] released in version 0.96.3
- [x] hap
  - [x] https://github.com/gap-packages/hap/pull/62 merged?
  - [x] released in version 1.37
- [x] homalg
  - [x] patch merged?
  - [x] released in version 2022.02-01
- [x] liering
  - [x] patch merged?
  - [x] released in version 2.4.2
- [x] MatricesForHomalg
  - [x] patch merged?
  - [x] released in version 2022.02-01
- [x] Modules
  - [x] patch merged?
  - [x] released in version 2022.02-01
- [x] polycyclic
  - [x] patch merged?
  - [x] released?
- [x] qpa
  - [x] https://github.com/gap-packages/qpa/pull/65 merged?
  - [x] released?
- [x] quagroup
  - [x] patch merged?
  - [x] released in version 1.8.3
- [x] semigroups
  - [x] https://github.com/semigroups/Semigroups/pull/798 merged? (changes were applied separately)
  - [x] released in version 4.0.0
- [x] SymbCompCC
  - [x] patch merged?
  - [x] released in version 1.3.2

**Comment by ThomasBreuer** [comment]
Concerning the idea to replace also `SUM` by `\+` etc., there are situations where names involving backslashes are not supported.
For example in the GAP-Julia integration, we can write `GAP.Globals.SUM`, but in order to use the name `\+`, one has to write something like `GAP.Globals.:+`.

**Comment by fingolfin** [comment]
Of course, but the people working on Julia related stuff (i.e., us) know this. I am not aware of other exceptions (although in general for the string forms, we prefer `"+"` over `"\+"`, but either works, so...)

**Comment by mohamed-barakat** [comment]
The `homalg_project` packages should be released in less than an hour from now. Thanks for the patches.
